projects/select-autocomplete/src/lib/select-autocomplete.module.ts

```diff
@@ -4,7 +4,8 @@ import { MatIconModule } from '@angular/material/icon';
 import { MatSelectModule } from '@angular/material/select';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-import { MatButtonModule, MatCheckboxModule } from '@angular/material';
+import { MatButtonModule } from '@angular/material';
+import { MatCheckboxModule } from '@angular/material';
```

Comment:
At least on Angular 9 my IDE complains if the imports aren't:
```typescript
import { MatButtonModule } from '@angular/material/button';
import { MatCheckboxModule } from '@angular/material/checkbox';
```

I guess it's pointless to comment as this repo seems abandoned.

Reply:
I agree ... I switched to another component I could use, meanwhile. :)

Reply:
> I agree ... I switched to another component I could use, meanwhile. :)

which one?

Reply:
Sorry that I missed your question ...
If it's still relevant, I used `ngx-mat-select-search`